lib/specifications/types/Library.js

```diff
@@ -1,9 +1,18 @@
-const fsPath = require("path");
-const posixPath = require("path").posix;
-const {promisify} = require("util");
-const resourceFactory = require("@ui5/fs").resourceFactory;
-const ComponentProject = require("../ComponentProject");
-
+import fsPath from "node:path";
+import posixPath from "node:path/posix";
+import {promisify} from "node:util";
+import ComponentProject from "../ComponentProject.js";
+import * as resourceFactory from "@ui5/fs/resourceFactory";
```

Comment:
only one method of the resourceFactory is used. Therefore use the named import not all.

Reply:
Done